db/migrate/20230505134353_remove_user_id_from_posts.rb

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+# Top-level documentation comment for RemoveUserIdFromPosts module
+class RemoveUserIdFromPosts < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :posts, :user_id, :bigint
+  end
```

Comment:
- [ ] @Uthmanbello great work removing user_id in your posts, but you forgot to add the author_id to make the association possible.

Reply:
Hello @danmainah ,

The `author_id` was added when I created the migration at first. I only added `user_id` in error that's why I removed it. If you check the table, you will find the `author_id` column there. Without it, the app wouldn't have worked.

Reply:
@Uthmanbello your schema indicates there is no author_id even after deleting it and running your migration files, the new generated schema shows the table as seen below 
![image](https://user-images.githubusercontent.com/61408860/236520380-2c872aeb-2972-4f0b-9c17-91f404031173.png)

Reply:
@danmainah ,
Below is my database:

![rails app screenshot(edit)](https://user-images.githubusercontent.com/112436398/236522382-ed85672e-79ae-4c24-8749-f89a7c2b0950.png)

Please, I need help trying to understand what is actually happening here.
Thanks.

Reply:
@danmainah ,

This is my schema here:

![schema screenshot](https://user-images.githubusercontent.com/112436398/236526495-05501b72-04e2-4a83-b397-724b6e4b98ea.png)